metrics/queries.go

```diff
@@ -272,22 +272,22 @@ func (m *NumericMetricExpr) ToSql(dialect Dialect) (string, error) {
 		return As(dialect.Count(Distinct(m.Column)), m.OutColumnAlias()).ToSql(dialect)
 
 	case METRIC_NUM_EMPTY:
-		return As(dialect.CountIf(Eq(m.Column, Int64(0))), m.OutColumnAlias()).ToSql(dialect)
+		return As(dialect.Coalesce(dialect.CountIf(Eq(m.Column, Int64(0))), Int64(0)), m.OutColumnAlias()).ToSql(dialect)
```

Comment:
adding default `0` to `count_if` and aggregations as they are able to return NULL